web/src/main/java/com/epam/rft/atsy/web/exceptionhandling/ErrorResponse.java

```diff
@@ -0,0 +1,45 @@
+package com.epam.rft.atsy.web.exceptionhandling;
+
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.HashMap;
+import java.util.Map;
+import lombok.AllArgsConstructor;
+import lombok.Value;
+
+/**
+ * This class stores error messages that can be sent back to the frontend. An
+ * instance of this class can hold a single error message providing general information
+ * about the error and separate error messages for each malformed field. In absence of errors
+ * the {@code errorMessage} is the {@code empty String} and the {@code fields} map is
+ * empty.
+ */
+@Value
+@AllArgsConstructor
+public class ErrorResponse {
```

Comment:
I think this should be renamed to RestResponse as it is returned when no error occurred.